e2e/images/README.md

```diff
@@ -1 +1,52 @@
-# rook-test-infrastructure
+# Rook Test Infrastructure
+
+The Rook Test Infrastructure project is a dockerfile
```

Comment:
it's awkward to say the project is a dockerfile. I mean, the project is more than a single file, right? You might say it's a docker project though